### PR TITLE
Update fb profile fields

### PR DESCRIPTION
**Pull request by loay**
connect to https://github.com/strongloop/loopback-component-passport/issues/147

### CODE REVIEW

**Comment by loay** [comment]
@superkhau @jannyHou PTAL. Thanks

**Comment by jannyHou** [comment]
LGTM if CI all pass 👍

File: README.md [hunk 1]
```diff
@@ -16,17 +16,7 @@ linking for LoopBack applications.
...
-this._profileURL = options.profileURL || 'https://graph.facebook.com/me';
-```
-
-with
-
-```
-this._profileURL = options.profileURL ||
-    'https://graph.facebook.com/v2.2/me?fields=first_name,gender,last_name,link,locale,name,timezone,verified,email,updated_time';
-```
+If more info is needed from the Facebook profile(email, gender, timezone, etc), it could still be obtained, see
```
Comment (superkhau) [inline README.md]: space after profile

Comment (superkhau) [inline README.md]: Maybe just change this whole sentence instead:

Facebook profile information (such as email, gender, timezone, etc) may still be included if necessary. See ...

**Pull request merged**